compair/tests/api/test_comparisons.py

```diff
@@ -501,46 +508,76 @@ def _submit_all_possible_comparisons_for_user(self, user_id):
             }
         }
 
+    """
+    The assumption of the test is not always true, especially when there are
```

Comment:
The rough assumption of this test should hold (answers that win more should have a higher score). Maybe write the test to capture this better instead of removing it.

Reply:
if there are time pressures, this test can probably be skipped for now though since the algorithm tests also test roughly the same thing

Reply:
Re-enabled the test. Changed the logic slightly to handle cases when answers have the same number of wins.